admin/modules/config/settings.php

```diff
@@ -1089,15 +1089,17 @@
 		{
 			$cache->update_statistics();
 		}
+		
```

Comment:
Can you remove the invisible characters from this line? Also, it's not necessary to open a new PR when a change is requested, just push changes to the same branch next time :)